SimFastTiming/FastTimingCommon/src/ETLElectronicsSim.cc

```diff
@@ -68,6 +90,11 @@ void ETLElectronicsSim::runTrivialShaper(ETLDataFrame &dataFrame,
       newSample.set(chargeColl[it] > adcThreshold_MIP_,false,tdc_time,adc);
       dataFrame.setSample(it,newSample);
 
+      //std::cout << it << " " << chargeColl[it] << " " << toa[it] << std::endl;
```

Comment:
delete commented-out code and extra lines

Reply:
Thank you, I missed this. Done.